Source/Xamarin/Prism.Forms/Ioc/RegisterByConventionAttribute.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Prism.Ioc
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    public class RegisterByConventionAttribute : Attribute
```

Comment:
Should probably be named something like RegisterForNavigationByConvention or AutoRegisterForNavigation. We don't want people thinking we are registering their services by convention too :)